epochStart/interface.go

```diff
@@ -9,14 +9,14 @@ import (
 
 // TriggerHandler defines the functionalities for an start of epoch trigger
 type TriggerHandler interface {
-	ForceEpochStart(round int64) error
+	ForceEpochStart(round uint64) error
 	IsEpochStart() bool
 	Epoch() uint32
 	ReceivedHeader(header data.HeaderHandler)
-	Update(round int64)
+	Update(round uint64)
 	EpochStartRound() uint64
 	EpochStartMetaHdrHash() []byte
-	Processed()
+	Processed(header data.HeaderHandler)
```

Comment:
The "Processed" name is not very suggestive.

Reply:
done